### PR TITLE
Align header elements; improve light/dark form controls

### PR DESCRIPTION
**Pull request by mitchray**
- Uniform height, font size and alignment of header elements
- Form control accent color takes on theme hue (see 'Random' checkbox)
- Dark mode form controls are now dark 🕶

### Light

Before

Screen Shot 2024-03-12 at 08 55 18

After

Screen Shot 2024-03-12 at 08 55 13

### Dark

Before

Screen Shot 2024-03-12 at 09 11 00

After

Screen Shot 2024-03-12 at 09 11 03

### CODE REVIEW

**Comment by lachlan-00** [comment]
pretty

File: public/themes/reborn/templates/dark.css [hunk 1]
```diff
@@ -48,12 +52,15 @@ hr {
...
 input[type=url],
 input[type=year],
+select,
 textarea {
-    background-color: #f5f5f5;
-    background-image: linear-gradient(to top, #eee, #fff);
-    border-color: #fff;
+    background-color: #000;
+    background-image: linear-gradient(to top, hsl(0, 0%, 8%), hsl(0, 0%, 4%));
+    border-color: #222;
```
Comment (lachlan-00) [inline public/themes/reborn/templates/dark.css:63]: should we make this light? / delete it?
image

Not sure what the fallback colour is but boxes can be a bit hard to see with 222 set

Comment (mitchray) [reply public/themes/reborn/templates/dark.css:63]: Give it

```
background-color: hsl(0, 0%, 10%);
background-image: linear-gradient(to top, hsl(0, 0%, 12%), hsl(0, 0%, 8%));
border-color: hsl(0, 0%, 30%);
```

Comment (lachlan-00) [reply public/themes/reborn/templates/dark.css:63]: yeah that looks heaps better, i'll add that

**Pull request merged**